app/views/pages/home.html.erb

```diff
@@ -5,27 +5,35 @@
-        <p>Your adviser will help you with:</p>
+        <p>Your adviser can help you:</p>
         <ul>
-          <li>getting school experience</li>
-          <li>funding your course</li>
-          <li>choosing the right training</li>
-          <li>writing your personal statement</li>
-          <li>interview tips</li>
+          <li>get school experience</li>
+          <li>fund your course</li>
+          <li>choose the right training</li>
+          <li>write your personal statement</li>
+          <li>find a teaching role</li>
+          <li>improve your subject knowledge</li>
         </ul>
         <h2 class="govuk-heading-m">Who can get an adviser?</h2>
 
         <p>To get an adviser, you'll need (one of the following):</p>
 
         <ul>
           <li>a bachelor's degree, class 2:2 (honours) or higher</li>
-          <li>to be studying for a degree and are predicted to get a class 2:2 (honours) or higher</li>
-          <li>an overseas qualification equivalent to a bachelor’s degree with honours, class 2:2 or higher</li>
+          <li>to be studying for a degree and predicted to get a class 2:2 (honours) or higher</li>
         </ul>
         
-        <p>If you’re thinking about returning to teaching, you'll need to have qualified teacher status (QTS) and want to teach a secondary subject in England to get an adviser.</p>
+        <p>If you’re thinking about returning to teaching, you'll need to have <a href="https://www.gov.uk/guidance/qualified-teacher-status-qts">qualified teacher status (QTS)</a> and want to teach a secondary subject in England to get an adviser.</p>
 
-        <p>Before you use this service as a non-UK citizen, you can check your qualifications by calling <%= link_to("0800 389 2500", "tel://08003892500") %>.</p>
+        <h2 class="govuk-heading-m">Non-UK trainees and teachers</h2>
+
+        <p>To get an adviser, you'll need to have or be studying for an overseas qualification of the same standard as a UK bachelor’s degree, class 2:2 (honours) or higher.</p>
+        
+        <p>You can check the UK equivalents of your degree qualifications by calling us on <%= link_to("0800 389 2500", "tel://08003892500") %>.</p>
+
+        <p>If you're already qualified as a teacher outside the UK, you'll need to have <a href="https://www.gov.uk/guidance/qualified-teacher-status-qts">English qualified teacher status (QTS)</a> and want to teach a secondary subject to get an adviser. You can <a href="https://apply-for-qts-in-england.education.gov.uk/eligibility/start">apply for English QTS</a> if you do not already have it.</p>
+          
+        <p>Even if you do not meet the eligibility requirements for this service, you can still teach in England. <a href="https://getintoteaching.education.gov.uk/non-uk-teachers">Find more information about teaching in England for non-UK citizens</a>.</p>
```

Comment:
```suggestion
        <p>Even if you do not meet the eligibility requirements for this adviser service, you can still <a href="https://getintoteaching.education.gov.uk/non-uk-teachers">teach in England as a non-UK citizen</a>.</p>
```

Is it worth incorporating the link into the first sentence to shorten the paragraph?